### PR TITLE
[master]Change active to be optional

### PR DESCRIPTION
**Pull request by trenxiaof**
Change active back to be optional

### CODE REVIEW

**Comment by trenxiaof** [comment]
@SeanHQF @zhshuj please help review.   along with https://github.ibm.com/zvc/icic-dev/pull/5447

File: zvmconnector/restclient.py [hunk 1]
```diff
@@ -887,7 +886,7 @@ def req_vswitch_set_vlan_id_for_user(start_index, *args, **kwargs):
         'request': req_guest_nic_uncouple_from_vswitch},
     'guest_create_network_interface': {
         'method': 'POST',
-        'args_required': 4,
+        'args_required': 3,
```
Comment (SeanHQF) [inline zvmconnector/restclient.py:889]: I saw this:
```
    'guest_register': {
        'method': 'POST',
        'args_required': 3,
        'args_optional': 1,
        'params_path': 1,
        'request': req_guest_register},
```
So why no 'args_optional': 1 here?

Comment (trenxiaof) [reply zvmconnector/restclient.py:889]: After checking the zcc API handling, I find most of the API using kwargs to pass the optional meter, not using args_optianal, like the api you added guest_nic_uncouple_from_vswitch. So I choose use this way.

**Review by SeanHQF — APPROVED** [review]

**Comment by SeanHQF** [comment]
@bjhuangr @jichenjc pls help review, thx

**Review by bjhuangr — APPROVED** [review]

**Pull request merged**